### ISSUE TITLE
please update version at least 3.7

### ISSUE THREAD

**Issue by xinbingyiran** [dotnetprojects/WpfExtendedToolkit#22]
v3.6 color picker properties are broken on windows 7.
v3.7 fixes the issue with ColorPicker in Windows 7. Here's the fix.
Go in file Xceed.Wpf.Toolkit/ColorPicker/Generic.xaml,
In the style targeting ""{x:Type local:ColorPicker}"",
Replace the Setter:
```<Setter Property="DropDownBackground" Value="{StaticResource PanelBackgroundBrush}" />```
by:
```
<Setter Property="DropDownBackground">
      <Setter.Value>
        <LinearGradientBrush EndPoint="0.5,1"
                              StartPoint="0.5,0">
          <GradientStop Color="#FFFAFBFB"
                        Offset="0" />
          <GradientStop Color="#FFF4F4F4"
                        Offset="1" />
        </LinearGradientBrush>
      </Setter.Value>
    </Setter>
```
issue is [here](https://github.com/xceedsoftware/wpftoolkit/issues/1536#issuecomment-538065029)

**Comment by jogibear9988** [comment]
could you create a pull request?

**Comment by xinbingyiran** [comment]
Is there no plan to upgrade this with a new version of the Xceed.Wpf.Toolkit?the Xceed.Wpf.Toolkit has update to 4.0+,and it has fix the bug in version 3.7

**Comment by jogibear9988** [comment]
no.
xceed changed licence, so there will be no more merges.
